src/utils/sentinel_1_reader.py

```diff
@@ -121,6 +121,10 @@ def _setup(rootpath, labelgeojson, npyfolder, min_area_to_ignore=1000):
         )
         labels = labels.loc[ignore]
         labels = labels.to_crs(crs)  # TODO: CHECK IF NECESSARY
+        labels["path"] = labels["fid"].apply(lambda fid: os.path.join(npyfolder, f"fid_{fid}.npz"))
```

Comment:
Hi, can I ask why you added this "path" because it's causing a crash in line 72 of s1_s2_reader.py?

`assert self.s1_reader.labels.equals(self.s2_reader.labels)
`
Documentation:

$ [(self.s1_reader.labels[each] == self.s2_reader.labels[each]).all() for each in self.s1_reader.labels.columns]
[True, True, True, True, True, True, False, True]

$ self.s1_reader.labels.columns
['fid', 'SHAPE_AREA', 'SHAPE_LEN', 'crop_id', 'crop_name', 'geometry',
       'path', 'exists']

$ self.s1_reader.labels['path'][0]
'/cmlscratch/izvonkov/tum-planet-radearth-ai4food-challenge/data/ref_fusion_competition_south_africa_train_source_sentinel_1/ref_fusion_competition_south_africa_train_source_sentinel_1_34S_19E_258N_asc_34S_19E_258N_2017/fid_80172.npz'

$ self.s2_reader.labels['path'][0]
'/cmlscratch/izvonkov/tum-planet-radearth-ai4food-challenge/data/ref_fusion_competition_south_africa_train_source_sentinel_2/ref_fusion_competition_south_africa_train_source_sentinel_2_34S_19E_258N_34S_19E_258N_2017/fid_80172.npz'

Reply:
run: `python train.py --competition south_africa --satellite s1_s2 --pos both_34 `

Reply:
Right, fixed!